lisa/operating_system.py

```diff
@@ -1417,31 +1417,22 @@ def _is_package_in_repo(self, package: str) -> bool:
         return 0 == result.exit_code
 
     def _get_information(self) -> OsInformation:
-        # The higher version above 7.0 support os-version.
-        try:
-            information = super(Fedora, self)._get_information()
-
-            # remove Linux Server in vendor
-            information.vendor = get_matched_str(
-                information.vendor, self.__vendor_pattern
-            )
-        except Exception:
-            # Parse /etc/redhat-release to support 6.x and 8.x. Refer to
-            # examples of __legacy_redhat_information_pattern.
-            cmd_result = self._node.execute(
-                cmd="cat /etc/redhat-release", no_error_log=True, expected_exit_code=0
-            )
-            full_version = cmd_result.stdout
-            matches = self.__legacy_redhat_information_pattern.match(full_version)
-            assert matches, f"cannot match version information from: {full_version}"
-            assert matches.group("vendor")
-            information = OsInformation(
-                version=self._parse_version(matches.group("version")),
-                vendor=matches.group("vendor"),
-                release=matches.group("version"),
-                codename=matches.group("codename"),
-                full_version=full_version,
-            )
+        # Parse /etc/redhat-release to support 6.x and 8.x. Refer to
```

Comment:
What's reason to make this change? The os-release should be standard, which works on new distros.

Reply:
@squirrelsc when run lis install case against openlogic centos 7_9 7.9.2022020700, the release is 7.0.0. read from os-release

Reply:
How about 8.x? If so, we can move 7.x to use redhat-release only.

Reply:
let me try